### PR TITLE
Upgrade Rector to 0.10.3

### PR DESCRIPTION
**Pull request by jmsche**

### CODE REVIEW

**Comment by jmsche** [comment]
@Guikingone ready for review

File: src/Scheduler.php [hunk 1]
```diff
@@ -163,7 +163,10 @@ public function getDueTasks(): TaskListInterface
             }
 
             if ($task->getExecutionStartDate() instanceof DateTimeImmutable) {
-                return $task->getExecutionStartDate() === $synchronizedCurrentDate || $task->getExecutionStartDate() < $synchronizedCurrentDate;
+                if ($task->getExecutionStartDate() === $synchronizedCurrentDate) {
+                    return true;
+                }
+                return $task->getExecutionStartDate() < $synchronizedCurrentDate;
```
Comment (Guikingone) [inline src/Scheduler.php:169]: Must be: 

```php
if ($task->getExecutionStartDate() === $synchronizedCurrentDate) {
    return true;
}

return $task->getExecutionStartDate() < $synchronizedCurrentDate;
```

Comment (jmsche) [reply src/Scheduler.php:170]: fixed

**Comment by Guikingone** [comment]
Looks good to me, ready to merge 👍🏻

**Pull request merged**